src/main/webapp/app/exercises/programming/manage/update/programming-exercise-test-schedule-date-picker.component.ts

```diff
@@ -17,9 +17,9 @@ import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
 export class ProgrammingExerciseTestScheduleDatePickerComponent implements ControlValueAccessor {
     @ViewChild('dateInput', { static: false }) dateInput: ElementRef;
     @Input() selectedDate: Date | null;
```

Comment:
can we also use undefined here? we would like to avoid null

Reply:
yes, changed it. null should not be used